### ISSUE TITLE
Training loss in my dataset shows nan

### ISSUE THREAD

**Issue by nashurgessa** [xuannianz/keras-fcos#28]
When I train in my own data, the loss shows me, nan. Any suggestion regarding this?

**Comment by matt-teng** [comment]
Hello, do you have some solutions to this? I also met this problem when I used it.

**Comment by matt-teng** [comment]
> When I train in my own data, the loss shows me, nan. Any suggestion regarding this?


Hello, do you have some solutions to this? I also met this problem when I used it.
